common/scripts/health-check.sh

```diff
@@ -86,27 +86,30 @@ function checking_operator_status() {
   output "${OC} -n ${namespace} get sub,ip,csv"
   ${OC} -n ${namespace} get sub,ip,csv | tee -a $logfile
   pending_sub=$(${OC} -n ${namespace} get sub -o=jsonpath='{.items[?(@.status.state=="UpgradePending")].metadata.name}')
-  echo $pending_sub | while read sub
-  do
-    installPlan=$(${OC} -n ${namespace} get sub ${sub} -o=jsonpath='{.status.installPlanRef.name}')
-    currentCSV=$(${OC} -n ${namespace} get sub ${sub} -o=jsonpath='{.status.currentCSV}')
-    approval=$(${OC} -n ${namespace} get ip ${installPlan} -o=jsonpath='{.spec.approval}')
-    approved=$(${OC} -n ${namespace} get ip ${installPlan} -o=jsonpath='{.spec.approved}')
-    if [[ "$approval" == "Manual" &&  "$approved" == "false" ]]; then
-      warning "InstallPlan ${installPlan} need approval, it will block operator automatic upgrade, run following command to approve this installPlan:"
-      output "oc -n ${namespace} patch installplan ${installPlan} -p '{"spec":{"approved":true}}' --type merge"
-      break
-    fi
-    output "Check subscription: ${OC} -n ${namespace} get sub ${sub} -oyaml"
-    ${OC} -n ${namespace} get sub ${sub} -oyaml | tee -a $logfile
-
-    output "Check csv: ${OC} -n ${namespace} get csv ${currentCSV} -oyaml"
-    ${OC} -n ${namespace} get csv ${currentCSV} -oyaml | tee -a $logfile
-
-    output "Check installPlan: ${OC} -n ${namespace} get ip ${installPlan} -oyaml"
-    ${OC} -n ${namespace} get ip ${installPlan} -oyaml | tee -a $logfile
-  done
-  [[ "X$pending_sub" == "X" ]] && success "All the common service operators is ready."
+  if [[ "X$pending_sub" == "X" ]]; then
+    success "All the common service operators is ready."
```

Comment:
All the common service operators is ready -> All the common service operators are ready